### PR TITLE
Openssl 1.1.1w

### PR DESCRIPTION
**Pull request by radetsky**
1. Updated OpenSSL to 1.1.1w 
2. Added experimental VisionOS support

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,6 +67,7 @@ endif
 .PHONY: build
 
 $(OUTPUT)/done.build:
+	@echo $(BUILD_FLAGS)
```
Comment (vixentael) [inline Makefile:70]: not needed?

File: create-openssl-framework.sh [hunk 2]
```diff
@@ -151,7 +151,7 @@ if [ $FWTYPE == "dynamic" ]; then
         elif [[ $PLATFORM == AppleTV* ]]; then
             MIN_SDK="-tvos_version_min $MIN_SDK_VERSION"
         elif [[ $PLATFORM == MacOSX* ]]; then
-            MIN_SDK="-macosx_version_min $MIN_SDK_VERSION"
+            MIN_SDK="-macos_version_min $MIN_SDK_VERSION"
```
Comment (vixentael) [inline create-openssl-framework.sh:154]: 👀

**Review by vixentael — COMMENTED** [review]
neat!

@G1gg1L3s if you have a moment, plz take a look

File: create-openssl-framework.sh (hunk 2 above)
Comment (radetsky) [reply create-openssl-framework.sh:154]: Yes, something changed. It works. I tested, and I will test again.

File: create-openssl-framework.sh [hunk 3]
```diff
@@ -187,6 +193,18 @@ if [ $FWTYPE == "dynamic" ]; then
...
+            -bitcode_bundle \
+            -lSystem \
+            -arch $ARCH \
+            $MIN_SDK \
+            -syslibroot $SDK \
+            -compatibility_version $COMPAT_VERSION \
+            -current_version $CURRENT_VERSION \
+            -application_extension \
+            -install_name $INSTALL_NAME \
+            -o $FWNAME.dylib"
```
Comment (G1gg1L3s) [inline create-openssl-framework.sh]: Leftover debug print?

File: create-openssl-framework.sh [hunk 4]
```diff
@@ -232,6 +250,7 @@ else
     NORMALIZE_OPENSSL_VERSION=no
 
     for SYS in ${ALL_SYSTEMS[@]}; do
+        echo bin/${SYS}*/lib/libcrypto.a bin/${SYS}*/lib/libssl.a
```
Comment (G1gg1L3s) [inline create-openssl-framework.sh]: Let's remove such prints if they are not functional. Or at least make them more understandable for the user.

**Review by G1gg1L3s — COMMENTED** [review]
Good job! Just a couple of places with debug prints.

**Review by vixentael — APPROVED** [review]

**Review by G1gg1L3s — APPROVED** [review]
Coool!

**Pull request merged**